### PR TITLE
Add categories to simulation settings dictionary

### PR DESCRIPTION
**Pull request by daniel-thom**

### CODE REVIEW

**Comment by daniel-thom** [comment]
Start by looking at PyDSS/defaults/simulation.toml.  Let me know what you think.  This should help us extend settings over time without having collisions.

File: PyDSS/dssInstance.py [hunk 1]
```diff
@@ -261,76 +261,88 @@ def RunStep(self, step, updateObjects=None):
...
             self._Logger.info('No post processing script selected')
-            self.postprocessor = None
 
         step = 0
         while step < Steps:
             self.RunStep(step)
-            if self.postprocessor is not None:
-                step = self.postprocessor.run(step, Steps)
+            for postprocessor in postprocessors:
+                step = postprocessor.run(step, Steps)
```
Comment (daniel-thom) [inline PyDSS/dssInstance.py]: @AadilLatif @akshaykj92  I'm not sure this code is doing the best thing. Future post-processors may want to run on time series simulations but not after every time step. Should `postprocessor.run` be moved to after all steps have been run?

File: PyDSS/pyPostprocessor/PostprocessScripts/AutomatedVoltageUpgrade.py [hunk 2]
```diff
@@ -32,68 +33,58 @@ class AutomatedVoltageUpgrade(AbstractPostprocess):
...
+
+        # TODO Akshay: this seems prone to error. Do we need a programmatic way of getting the filename?
+        thermal_filename = "Thermal_upgrades_pen_0.dss"
+        thermal_dss_file = os.path.join(
+            project.get_post_process_directory(self.config["Thermal scenario name"]),
+            thermal_filename
+        )
+        if not os.path.exists(thermal_dss_file):
+            raise InvalidParameter(f"AutomatedThermalUpgrade did not produce thermal_filename")
+        # TODO Akshay: redirect to the above file where appropriate
```
Comment (daniel-thom) [inline PyDSS/pyPostprocessor/PostprocessScripts/AutomatedVoltageUpgrade.py]: @akshaykj92 Note the two comments here.  You're welcome to push code to this branch or wait until we merge to the develop branch.

File: PyDSS/dssInstance.py (hunk 1 above)
Comment (AadilLatif) [reply PyDSS/dssInstance.py]: That can easily be done in the postprocess script, which is why i pass both  step and Steps. I have built a couple of small scripts to make sure they work.

**Pull request merged**